roles/grafana/templates/grafana.ini.j2

```diff
@@ -212,3 +212,12 @@ provider = {{ grafana_image_storage.provider }}
 {{ k }} = {{ v }}
 {%   endfor %}
 {% endif %}
+
+# Feature toggles
+{% if grafana_feature_toggles != {} %}
+[feature_toggles]
```

Comment:
Minor nitpick, but can you please move the comment inside the condition? Without it you will end up with a out of place comment if you don't have any feature toggles defined.


```suggestion
{% if grafana_feature_toggles != {} %}
# Feature toggles
[feature_toggles]
```

Reply:
Added your change